kqueen/engines/jenkins.py

```diff
@@ -47,9 +47,26 @@ class JenkinsEngine(BaseEngine):
                 'validators': {
                     'required': True
                 }
+            },
+            # TODO below form should be increased dynamically in case of 'add one more parameter'
+            # Use-case: 'add parameter'->'key': 'value'-> 'add parameter'-> etc...
+            # Should return dict{}
+            'override_parameters': {
```

Comment:
please, replace with
            'override_parameters': {
                'type': 'parameters',
                'label': 'Jenkins job parameters',
                'order': 2,
                'validators': {
                          'required': False
                }
            }
        },
        'cluster': {}
    }

Also, did you delete clusters occasionally?